src/state/timer-state.js

```diff
@@ -166,6 +172,28 @@ class TimerState {
     this.publishConfig()
   }
 
+  setClearClipboardHistoryBetweenTurns(value) {
+    this.clearClipboardHistoryBetweenTurns = value
+    this.publishConfig()
+  }
+
+  clearClipboardHistory() {
+    const millisecondsNeededBetweenWrites = 180
+    const numberOfItemsHistoryStores = 25

```

Comment:
What is the significance of these magic numbers? Does it differ based on OS or other extensions / tooling that are installed? Should the number of items be configurable?

Reply:
These are specific to the Windows clipboard history tool, and at least `numberOfItemsHistoryStores` is almost certainly different on different tools (unsure about `millisecondsNeededBetwenWrites`)
I could make one or both of those configurable, in my experience these numbers aren't easy to find without a little googling, so the user may not know what number belongs there without a little digging.